maakay-bot.py

```diff
@@ -623,16 +624,14 @@ async def tournament_all(ctx):
     await ctx.defer(hidden=True)
     discord_user, created = await sync_to_async(User.objects.get_or_create)(discord_id=str(ctx.author.id))
 
-    embed = discord.Embed(title="Active Tournaments")
+    embed = discord.Embed(title="Active Tournaments", color=Color.orange())
+    embed.set_author(name=ctx.author.name, icon_url=ctx.author.avatar_url)
 
     if Tournament.objects.filter(Q(hosted_by=discord_user), Q(status=Tournament.ONGOING)).exists():
 
         tournaments = (await sync_to_async(Tournament.objects.filter)(Q(hosted_by=discord_user), Q(status=Tournament.ONGOING))).order_by('-created_at')[:5]
         for tournament in tournaments:
-            embed.add_field(name="ID", value=tournament.uuid_hex, inline=False)
-            embed.add_field(name="Title", value=tournament.title)
-            embed.add_field(name="Description", value=tournament.description)
-            embed.add_field(name="Reward (TNBC)", value=convert_to_decimal(tournament.amount))
+            embed.add_field(name=f"**{tournament.title}**\n *{tournament.description}*", value=f">Role: Host\n >Amount: {tournament.amount}", inline=False)
```

Comment:
same here. Also need to make sure the amount is in decimals.
`embed.add_field(name=f"**{tournament.title}**\n *{tournament.description}*", value=f"> Role: Host\n > Amount: {convert_to_decimal(tournament.amount)}", inline=False)`